Scribe-i18n/scripts/iOS/convert_xcstrings_to_jsons.py

```diff
@@ -9,17 +9,34 @@
 import json
 import os
 
+# Determine the directory
 directory = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-file = open(os.path.join(directory, "Localizable.xcstrings"), "r").read()
+
+# Read the Localizable.xcstrings file
+try:
+    with open(os.path.join(directory, "Localizable.xcstrings"), "r") as f:
+        file = f.read()
+except FileNotFoundError:
+    print("Error: Localizable.xcstrings file not found.")
+    exit(1)
+
 dir_list = os.listdir(directory)
 languages = [file.replace(".json", "") for file in dir_list if file.endswith(".json")]
 
+# Ensure the destination directory exists
+if not os.path.exists(directory):
```

Comment:
This check should probably go before `dir_list` is created on line 23, since it already uses the directory. I don't think we need to check `directory` itself, since that's supposed to be the root `Scribe-i18n` directory anyway. It could be worth it if there's a variable for the `jsons` directory and you do the same check on that instead.